docs/extensions/bundled/introduction.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Introduction
+level: advanced
+---
+Introduction
+============
+
+Bundled extensions, or "Bundles", should be thought of as _components_ that are
+_specific_ to **your site project(s)**. As such, developed, managed, and
+deployed using the same tools and methodologies you would deploy your Bolt site
+would normally.
+
+Writing a Bundle has a lot in common with writing a regular extension, such as
+you'd find, and install, from the [Market Place][market]. In that, as simple as
+creating a class in your Application that implements
+`Bolt\Extension\ExtensionInterface`.
+
+However, it is very important to note that, Bundles **do not**:
```

Comment:
No, that one is correct. The point is "However Bundles", what is in between is an interjection.